nssp/delphi_nssp/run.py

```
@@ -72,18 +72,21 @@ def run_module(params):
         Nested dictionary of parameters.
     """
     start_time = time.time()
-    logger = get_structured_logger(
-        __name__,
-        filename=params["common"].get("log_filename"),
-        log_exceptions=params["common"].get("log_exceptions", True),
-    )
+    issue_date = params.get("patch", {}).get("current_issue", None)
+    source_dir = params.get("patch", {}).get("source_dir", None)
```

We should throw a error and exit the program if there isn't a source_dir